pdns/dnssecsigner.cc

```diff
@@ -50,6 +50,17 @@ static std::string getLookupKey(const std::string& msg)
   }
 }
 
+static std::string hashPublicKey(const std::string& pubKey)
+{
+  /* arbitrarily cut off at 64 bytes, the main idea is to save space
+     for very large keys like RSA ones (1024+ bytes) by storing a 20 bytes hash
```

Comment:
```suggestion
     for very large keys like RSA ones (128+ bytes) by storing a 20 bytes hash
```

Reply:
Just a nitpick: perhaps this function is better in the `pdns` namespace as `cutOffOrHash(nBytes, text)`?